lnrpc/rpc.proto

```diff
@@ -733,10 +733,8 @@ service Lightning {
     };
 
     /**
-    VerifyChanBackup allows a caller to verify the integrity of a channel
-    backup snapshot. This method will accept both a packed Single, and also a
-    Packed multi. Two bools are returned which indicate if the passed Single
-    (if present) is valid and also if the passed Multi (if present) is valid.
+    VerifyChanBackup allows a caller to verify the integrity of a channel backup
+    snapshot. This method will accept either a packed Single or a Packed multi.
```

Comment:
comment should mention that specifying both will return an error

Reply:
Fixed.